codegen/src/test/resources/software/amazon/awssdk/codegen/poet/model/alltypesrequest.java

```diff
@@ -1064,6 +1065,21 @@ public String toString() {
         Builder structWithNestedTimestampMember(StructWithTimestamp structWithNestedTimestampMember);
 
         /**
+         * Sets the value of the StructWithNestedTimestampMember property for this object.
+         *
+         * This is a convenience which creates an instance of the {@link StructWithTimestamp.Builder} avoiding the need
```

Comment:
Include something on how we will call build automatically?

Reply:
Done